src/connector/src/nexmark/split.rs

```diff
@@ -47,4 +47,12 @@ impl NexmarkSplit {
             start_offset,
         }
     }
+
+    pub fn update(&self, start_offset: String) -> Self {
```

Comment:
```suggestion
    pub fn update(&mut self, start_offset: String) {
```

I think the natural semantic of "update" is `&mut self`, or it should be named "copy_with_offset()" etc,.

Reply:
I prefer `copy_with_offset` or may get some problem with copy trait. will fix